<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Open</class>
 <widget class="QDialog" name="Dialog_Open">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Datei Ã–ffnen</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Icons/File_Open.png</normaloff>:/Icons/File_Open.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{color:rgb(160,160,160);
                  background-color: rgba(0, 0, 0,128);
                  border-radius:3px;}
QLineEdit:focus{color:rgb(192,192,192);
                             border: 1px solid;
                             border-color: rgb(0, 128, 0);
                            selection-background-color: rgb(40, 76, 117);}

QListView {color:rgb(160,160,160);
                  background-color: rgba(0,0,0,128);
                  border-radius:5px;
      			 /*show-decoration-selected: 1; /* make the selection span the entire width of the view */
}

QListView:focus{color:rgb(192,192,192);
                             /*border: 1px solid;
                             border-color: rgb(0, 128, 0);*/
                            selection-background-color: rgb(40, 76, 117);}

/*QListView::item:alternate {
    background: #EEEEEE;
}*/

QListView::item:selected {
	 background: rgb(128,128,192);
    /*border: 1px solid #6a6ea9;*/
}

QListView::item:selected:!active {
	 background: rgb(128,128,192);
    /*background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #ABAFE5, stop: 1 #8588B2);*/
}

QListView::item:selected:active {
	 background: rgb(128,128,192);
    /*background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #6a6ea9, stop: 1 #888dd9);*/
}

QListView::item:hover {
    color:rgb(255,255,255);
    background: rgb(128,192,128);
}

QScrollBar:vertical {
    border: 1px solid #999999;
    background: rgba(255, 255, 255,32);
    width: 12px; /* Breite der Scrollbar */
    margin: 0px 0px 0px 0px;
    border-radius: 6px; /* Abrundung der Scrollbar-Leiste */
}

QScrollBar::handle:vertical {
    background: rgb(128,128,192);/*#a0a0a0; /* Farbe des Scrollbar-Griffs */
    min-height: 20px;
    border-radius: 5px; /* Abrundung des Griffs */
}

QScrollBar::add-line:vertical {
    border: none;
    background: none;
}

QScrollBar::sub-line:vertical {
    border: none;
    background: none;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
    border: none;
    background: none;
    color: none;
}

/*QToolButton{background-color: rgba(0, 0, 0,0);}*/</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit_Project"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_Tags">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/Icons/Project/tag.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_Tool">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/Icons/Project/tools.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListView" name="listView_Project">
       <property name="maximumSize">
        <size>
         <width>290</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="RoundedLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RoundedLabel</class>
   <extends>QLabel</extends>
   <header>Classes/roundedlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_Open</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_Open</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
